src/renderer/components/layout/sidebar-item.tsx

```diff
@@ -76,19 +83,7 @@ export class SidebarItem extends React.Component<SidebarItemProps> {
         </NavLink>
         {isExpandable && expanded && (
           <ul className={cssNames("sub-menu", { active: isActive })}>
-            {subMenus.map(({ title, routePath, url = routePath }) => {
-              const subItemId = `${id}${routePath}`;
-
-              return (
-                <SidebarItem
-                  key={subItemId}
-                  id={subItemId}
-                  url={url}
-                  text={title}
-                  isActive={isActiveRoute({ path: url, exact: true })}
```

Comment:
Shouldn't it be possible to just remove this line and fix the bug?

Reply:
Dunno. CDS's and 3rd level menus are handled a bit differently.